.github/workflows/ci.yml

```diff
@@ -1,21 +1,21 @@
 name: CI
 
-on: [push, pull_request]
+on: [push]
```

Comment:
probably better to run the report on each push to master rather than on any push to any branch?